### PR TITLE
macOS day 5 survey

### PR DESCRIPTION
**Pull request by samsymons**
Task: https://app.asana.com/0/1207619243206445/1208279078919897/f

This PR adds the macOS day 5 survey. This is a draft PR until confirming the survey contents and behaviour.

**How to test:**

Before you start, you'll need to be using English as your device's language - this survey targets en-US, en-CA, en-GB, and en-AU.

1. Update `RemoteMessagingClient.swift`'s debug URL to use the [macOS staging URL](https://staticcdn.duckduckgo.com/remotemessaging/config/staging/macos-config.json)
2. Next, we need to simulate having installed the app between 5 and 8 days ago; to do this, you'll need to drop a local copy of BSK onto your project so that you can edit it, open UserAttributeMatcher.swift, then look for the line case let matchingAttribute as DaysSinceInstalledMatchingAttribute. Edit the logic inside this case to change the value of daysSinceInstall to between 5-8. _(Note: we have an option for changing the install date in the debug menu, but I think it affects a key that is separate from the statistics store - need to look into it further)_
3. Launch the app, open the Debug menu's Remote Messaging Config section, select "Reset Remote Messages", then select "Refresh Messages" in the same Debug menu
4. Open a new tab and the message should appear - click the survey URL and make sure it opens as expected, and that parameters look sensible

### CODE REVIEW

File: live/macos-config/macos-config.json [hunk 1]
```diff
@@ -101,6 +119,15 @@
...
       }
+    },
+    {
+      "id": 6,
+      "attributes": {
+        "daysSinceInstalled": {
+          "min": 5,
+          "max": 8
+        }
+      }
```
Comment (samsymons) [inline live/macos-config/macos-config.json:130]: This will need to be updated to target English users.

File: live/macos-config/macos-config.json [hunk 2]
```diff
@@ -101,6 +119,26 @@
           "value": ["privacy_pro_survey_1"]
         }
       }
+    },
+    {
+      "id": 6,
+      "attributes": {
+        "appVersion": {
+          "min": "1.106.0"
```
Comment (samsymons) [inline live/macos-config/macos-config.json:127]: This is the current internal version, and is the release that removes the previous survey support. With this version check in place, I plan to merge this PR as soon as it's approved, and it'll start being visible to users next week as the release rolls out.

**Comment by samsymons** [comment]
⚠️ I've opened a second macOS RMF PR which means that the copy in this PR will no longer be on the staging URL, sorry about that. I'd recommend using jsonblob.com instead; I can set up a copy of this on Wednesday and update the instructions.

**Review by SabrinaTardio — APPROVED** [review]
Works as expected! thanks!

**Pull request merged**